docs/GettingStarted.md

```diff
@@ -101,3 +101,39 @@ Windows requires that one of the following criteria be met prior to loading a dr
 3. The OS is running in [test-signing mode](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/the-testsigning-boot-configuration-option), the [driver is test signed](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/test-signing-a-driver-through-an-embedded-signature) and the [test certificate is installed](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/installing-test-certificates).
 
 Official releases of eBPF for Windows will be production signed.
+
+## Tests in Ebpf-For-Windows
+
+The tests in Ebpf-For-Windows are written using the [Catch2](https://github.com/catchorg/Catch2) test framework.
+
+There are 2 types of tests present:
+1.	End-to-End tests - These exercise the entire path: User API, verifier, jitter, execution context and mock extensions.
+2.	Unit tests – These exercise a specific component in isolation.
+
+### end_to_end.exe
+This test uses a mocking layer to bind the user mode components to the kernel mode
+components via a Mock IOCTL interface. The tests initialize the user mode and kernel
+mode components, load a eBPF program from an ELF file and then run the eBPF program
+by having the mocked extensions emit events.
+
+### platform_unit_test.exe
+This test invokes API’s exposed by the user-mode implementation of platform
```

Comment:
```suggestion
This test invokes APIs exposed by the user-mode implementation of platform
```
FYI: https://abbreviations.yourdictionary.com/simple-rules-for-plural-acronyms-and-abbreviations.html